lib/bitstyles_phoenix.ex

```diff
@@ -33,4 +33,22 @@ defmodule BitstylesPhoenix do
   showcase("Inline errors", """
     ui_error_tag("Some error")
   """)
+
+  showcase_section("Badge", "Components.Badge.ui_badge/2")
+
+  showcase("Gray", """
+    ui_badge("Published")
```

Comment:
This doesn’t work, because there is no `ui_badge/1`. How can I make this work, and default to `variant: "gray"`? 🤔

Reply:
You only defined ui_badge/2.

You probably want to define a `ui_badge/1` like this:
```
def ui_badge(label), do: ui_badge(label, [])
```

Normally it would also work with default args, but you can't define default args in this case (with the extra `do` definition) I think.

Reply:
Sorry 🙈 updated